source/ftrack_api/accessor/server.py

```diff
@@ -107,7 +107,7 @@ def _write(self):
         self.seek(0)
 
         # Put the file based on the metadata.
-        response = requests.put(
+        response = self._session._request.put(
```

Comment:
Would this not mean that we now pass headers and cookies to a third party? Say S3

Reply:
Discussed, lets do the same as in the event hub. Use request directly and pass headers/cookies.

Reply:
Done.